Frontend/ui-driver/src/Resource/Localizable/EN.purs

```diff
@@ -184,7 +184,7 @@ getEN stringKey =
         EMPTY_RIDES -> "Empty Rides"
         YOU_HAVE_NOT_TAKEN_A_TRIP_YET -> "You haven't taken a trip yet"
         BOOK_NOW -> "Book Now"
-        RESEND_OTP_IN -> "Resend OTP in "
+        RESEND_OTP_IN -> "<u>Resend OTP in "
```

Comment:
closing tag for u?

Reply:
We need to add underline for timer values also so no closing tag.